src/spy-performance/spy-functions/spy-selector.ts

```diff
@@ -31,25 +33,25 @@ const spySelectorMemoizer = (originalMemoized: any) => {
   };
 };
 
-export const spyCreateSelectorTime = function (originalCreateSelectorFunc: any) {
-  return function () {
-    arguments[0] = spySelectorMemoizer(arguments[0]);
-    const spiedFunction = originalCreateSelectorFunc(...arguments);
+export const spyCreateSelectorTime = function (originalCreateSelectorFunc: GenericFunction) {
+  return (...args: any[]) => {
```

Comment:
TS does NOT play nice with `arguments`.

The alternative would be to add `@ts-ignore` in certain places.